docs/unified-test-documentation/test-matrix.md

```diff
@@ -29,6 +31,12 @@ described in this documentation are dedicated to the given platform
 | 1.   | TPM Support                                       | TPM           | TPM001.001                           |
 | 2.   | Verified Boot support                             | VBO           | VBO001.001, VBO002.001, VBO003.001   |
 
+### Module: Dasharo performance
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | Coreboot boot measure                             | CBMEM         | All                                  |
```

Comment:
```suggestion
| 1.   | coreboot boot measure                             | CBMEM         | All                                  |
```
@macpijan can we have pre-commit for such corrections?

Reply:
All occurrences of the word `coreboot` have been improved in test-matrix file.